From 90202c13466fcde28fd56d349a3380d68dbda5f2 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Tue, 22 May 2012 15:05:18 -0400
Subject: [PATCH] libquota: add missing time.h include

This code uses time() but doesn't include time.h leading to:
quotaio.c:89:2: warning: implicit declaration of function 'time'

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 lib/quota/quotaio.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/lib/quota/quotaio.c b/lib/quota/quotaio.c
index 481d5f5..6e917f0 100644
--- a/lib/quota/quotaio.c
+++ b/lib/quota/quotaio.c
@@ -11,6 +11,7 @@
 #include <string.h>
 #include <unistd.h>
 #include <stdlib.h>
+#include <time.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/file.h>
-- 
1.7.8.6

